src/client/components/table-of-contents/readme.md

```diff
@@ -1 +1,12 @@
 # Table of contents
+
+- Lists all sections on a page
+- Links to each section
+- Has a scroll spy
```

Comment:
Since the `readme` is client oriented, it would be nice to add a short explanation on the advantage, e.g. `has a scroll spy to indicate which section is currently visible on the user's screen` and `is sticky, so it's always visible`.